### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by Neophen**
There is an issue when trying to use this with vite-ssg:
```
[vite-ssg] An internal error occurred.
[vite-ssg] Please report an issue, if none already exists: https://github.com/antfu/vite-ssg/issues
file:///Users/mykolas/Documents/Projects/the_cheat/.vite-ssg-temp/main.mjs:29
import hljsVuePlugin from "@highlightjs/vue-plugin";
       ^^^^^^^^^^^^^
SyntaxError: The requested module '@highlightjs/vue-plugin' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:127:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:193:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
    at async build (/Users/mykolas/Documents/Projects/the_cheat/node_modules/.pnpm/vite-ssg@0.17.10_2a1f4eb68d7efe0f1402d91afc414ace/node_modules/vite-ssg/dist/node/cli.cjs:180:87)
    at async Object.handler (/Users/mykolas/Documents/Projects/the_cheat/node_modules/.pnpm/vite-ssg@0.17.10_2a1f4eb68d7efe0f1402d91afc414ace/node_modules/vite-ssg/dist/node/cli.cjs:314:3)
```

Adding these exports fixes it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,14 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "publish": "npm publish --access public --tag next"
   },
+  "exports": {
+    ".": {
+      "require": "./dist/highlightjs-vue.min.js",
+      "import": "./dist/highlightjs-vue.esm.min.js",
+      "types": "./dist/vue.d.ts"
+    },
+    "./*": "./*"
```
Comment (joshgoebel) [inline package.json:19]: Why is this line necessary?

Comment (smarlhens) [reply package.json:19]: [nodejs doc](https://nodejs.org/api/packages.html#package-entry-points): 

> As a last resort, package encapsulation can be disabled entirely by creating an export for the root of the package "./*": "./*". This exposes every file in the package at the cost of disabling the encapsulation and potential tooling benefits this provides. 

I'm not sure to understand why we need it, @Neophen can you elaborate ?

Comment (Neophen) [reply package.json:19]: Sorry that is not neccasary :D

Comment (smarlhens) [inline package.json:19]:
```suggestion
    }
```

**Review by smarlhens — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (joshgoebel) [reply package.json:19]: Let's remove i& it's not necessary.
